cluster/images/hyperkube/master-multi.json

```diff
@@ -27,9 +35,22 @@
               "--address=0.0.0.0",
               "--etcd-servers=http://127.0.0.1:4001",
               "--cluster-name=kubernetes",
-              "--admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota",
-              "--v=2"
-        ]
+              "--admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota",
```

Comment:
Why? I removed it on request by @pmorie 
Discussion here: https://github.com/kubernetes/kubernetes/pull/17213#discussion_r44854820